app/src/main/res/layout/loyalty_card_edit_activity.xml

```diff
@@ -368,12 +368,21 @@
                                   android:padding="10.0dip"
                                   android:layout_width="fill_parent"
                                   android:layout_height="wrap_content"
-                                  android:id="@+id/barcodeCaptureLayout">
+                                  android:id="@+id/barcodeCaptureLayout"
+                                  android:baselineAligned="false">
```

Comment:
If I understand correctly from 

https://medium.com/@pareshmayani/androiddev-tip-set-android-baselinealigned-false-on-this-element-for-better-performance-810fa3148043

this was likely necessary to get the buttons to align up correctly, correct?

Reply:
Yes. The new button spans two lines on narrower devices and the default is to align by text, not by their bounding boxes.